src/common/user-store.ts

```diff
@@ -136,6 +112,24 @@ export class UserStore extends BaseStore<UserStoreModel> {
     return this.shell || process.env.SHELL || process.env.PTYSHELL;
   }
 
+  startMainReactions() {
```

Comment:
Having this code not be part of either `load` or `constructor` means that we can ignore uncancellable reactions in unit tests.